common-docs/identity/sign-in.md

```diff
@@ -30,6 +42,20 @@ It is possible! Do you play Minecraft, game on an Xbox, subscribe to Office 365,
 
 No. You can create an MSA using your current email address if it is not hosted by Microsoft (e.g.: Gmail, Yahoo Mail, and others). When creating your MSA account, just enter your existing email address.
 
+Create a Microsoft Account using your current email address here https://account.microsoft.com/account.
+
 ## Can I use my school or work account?
 
 If your school or employer's email system is managed by Microsoft Azure Active Directory, then generally it should work in MakeCode subject to restrictions put in place by your IT dept.
+
+## When I sign in, what information does MakeCode collect about me?
+
+The privacy and safety of our users is very important to us. Microsoft MakeCode adheres to Microsoft's Privacy Policy https://makecode.com/privacy. We do not collect or use personal data beyond that needed to operate and support MakeCode features and services.
+
+## How can I completely remove my data and account information?
```

Comment:
To avoid confusion with MSA/AAD accounts, we've been asked not to use the word "account" when referring to a MakeCode profile. We can use "profile" or "user profile".